web_developer_mode_optional/static/description/icon.svg

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@legalsylvain @pedrobaeza I just thought it could be useful for anyone that could modify the module to have the vectorial source of the image. But I can delete, if needed, of course. Should I take the "no .svg" as an OCA policy for any future module?